examples/claud_example.py

```diff
@@ -0,0 +1,46 @@
+import asyncio
+import os
+from os import getenv
+
+import anthropic
+from dotenv import load_dotenv
+from pydantic import BaseModel
+
+from gpt_json import GPTJSON, GPTMessage, GPTMessageRole
+from gpt_json.models import GPTModelVersion
+
+load_dotenv()
+API_KEY = getenv("ANTHROPIC_API_KEY")
+
+
+class SentimentSchema(BaseModel):
+    sentiment: str
+
+
+PROMPT_TEMPLATE = """
+Analyze the sentiment of the given text.
+
+Respond with the following JSON schema:
+
+{json_schema}
+
+Text: I love this product. It's the best thing ever!
+"""
+
+
+async def runner():
+    gpt_json = GPTJSON[SentimentSchema](API_KEY, model=GPTModelVersion.CLAUD)
```

Comment:
Can you stub out what the expected output format of the anthropic API is, and write a bit of coverage for this in unit tests?

Reply:
Sure. I can do that later this week